tests/testTrajectory.cpp

```diff
@@ -105,11 +105,11 @@ TEST(Trajectory, error) {
   // steps = 2+3 per walk cycle, 5 legs involved
   const size_t expected = repeat * ((2 + 3) * 5);
   EXPECT_LONGS_EQUAL(expected, contact_link_objectives.size());
-  // regression
-  auto last_factor = boost::dynamic_pointer_cast<PointGoalFactor>(
-      contact_link_objectives.back());
-  EXPECT(gtsam::assert_equal(gtsam::Point3(-0.190001, -0.300151, 0.000151302),
-                             last_factor->goalPoint(), 1e-5));
+  // // regression
+  // auto last_factor = boost::dynamic_pointer_cast<PointGoalFactor>(
```

Comment:
???

Reply:
As we rewrite the PointGoalFactor with expressions, we won't be able to access the member function goalPoint()